docs/workflow/building/coreclr/nativeaot.md

```diff
@@ -108,6 +108,13 @@ and the following two references to the project file itself:
 ```
 You should now be able to publish the project for Wasm: `dotnet publish --self-contained -r browser-wasm /p:MSBuildEnableWorkloadResolver=false`. This produces `YourApp.html` and `YourApp.js` files under `bin\<Config>\<TFM>\browser-wasm\native`. The former can be opened in the browser, the latter - run via NodeJS.
 
+### WASI
+To build with WASI, 
+- Download and install the Wasi SDK from https://github.com/WebAssembly/wasi-sdk/releases (only Windows is supported currently)
+- Set the WASI_SDK_PATH to the location where it is installed, e.g. set WASI_SDK_PATH=c:\github\wasi-sdk
+-  Run `build clr.nativeaotruntime+clr.nativeaotlibs+libs -c [Debug|Release] -a wasm -os wasi`. This will create the architecture-dependent libraries needed for linking and runtime execution, as well as the managed binaries to be used as input to ILC.
+- If you want to test the packages then they can be built with `build nativeaot.packages -a wasm -os wasi`
```

Comment:
It would be better to integrate these bits into the text that talks about cloning EMSDK, e. g.:
```
- [Existing] Clone the [emsdk](https://github.com/emscripten-core/emsdk) repository...
- [New] To build for WASI, download and install the WASI SDK...
- [Existing] Run build clr.nativeaotruntime+clr.nativeaotlibs+libs -c [Debug|Release] -a wasm -os [browser|wasi]
```

Reply:
Done, thanks